src/Jackett.Common/Definitions/nekobt.yml

```diff
@@ -0,0 +1,250 @@
+    label: MTL Subtitles
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Include Only"
+      "false": "Exclude"
+  - name: group_id
+    type: text
+    label: Group ID
+    default: ""
+  - name: group_primary
+    type: select
+    label: Primary Group Uploader
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Yes"
+      "false": "No"
+  - name: group_secondary
+    type: select
+    label: Secondary Group Uploader
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Yes"
+      "false": "No"
+  - name: group_childs
+    type: select
+    label: Include Child Groups
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Yes"
+      "false": "No"
+  - name: group_parents
+    type: select
+    label: Include Parent Groups
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Yes"
+      "false": "No"
+  - name: uploader_id
+    type: text
+    label: Uploader ID
+    default: ""
+  - name: uploader_uploads
+    type: select
+    label: Uploader is Primary
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Yes"
+      "false": "No"
+  - name: uploader_contributions
+    type: select
+    label: Uploader Contributions Only
+    default: ""
+    options:
+      "": "No Filter"
+      "true": "Yes"
+      "false": "No"
+
+search:
+  paths:
+    - path: api/torznab/api
+      response:
+        type: xml
+
+  inputs:
+    t: "search"
+    q: "{{ .Keywords }}"
+    apikey: "{{ .Config.apikey }}"
+    tvdbid: "{{ .Query.TVDBID }}"
+    season: "{{ .Query.Season }}"
+    ep: "{{ .Query.Ep }}"
+    levels: "{{ .Config.levels }}"
+    video_codec: "{{ .Config.video_codec }}"
+    video_type: "{{ .Config.video_type }}"
+    audio_lang: "{{ .Config.audio_lang }}"
+    fansub_lang: "{{ .Config.fansub_lang }}"
+    sub_lang: "{{ .Config.sub_lang }}"
+    hardsub: "{{ .Config.hardsub }}"
+    otl: "{{ .Config.otl }}"
+    mtl: "{{ .Config.mtl }}"
+    group_id: "{{ .Config.group_id }}"
+    group_primary: "{{ .Config.group_primary }}"
+    group_secondary: "{{ .Config.group_secondary }}"
+    group_childs: "{{ .Config.group_childs }}"
+    group_parents: "{{ .Config.group_parents }}"
+    uploader_id: "{{ .Config.uploader_id }}"
+    uploader_uploads: "{{ .Config.uploader_uploads }}"
+    uploader_contributions: "{{ .Config.uploader_contributions }}"
+
+  rows:
+    selector: rss > channel > item
+
+  fields:
+    title:
+      selector: title
+
+    details:
+      selector: comments
+
+    download:
+      selector: enclosure
+      attribute: url
+
+    date:
+      selector: pubDate
+
+    category:
+      selector: '[name="category"]'
+      attribute: value
+
+    size:
+      selector: size
+
+    seeders:
+      selector: '[name="seeders"]'
+      attribute: value
+
+    leechers:
+      selector: '[name="leechers"]'
+      attribute: value
+
+    grabs:
+      selector: '[name="grabs"]'
+      attribute: value
+
+    infohash:
```

Comment:
Probably better to use `magnet` with the provided `magneturl` here, as it uses their own tracker, along with other relevant ones